app/views/RoomsListView/index.js

```diff
@@ -153,7 +154,7 @@ export default class RoomsListView extends React.Component {
 	componentDidMount() {
 		this.getSubscriptions();
 		const { navigation, userLanguage } = this.props;
-		moment.locale(userLanguage);
+		moment.locale(toMomentLocale(userLanguage));
```

Comment:
Why on `RoomsListView`? 🤔 
Will it work on `OnboardingView`, for example?

Reply:
fixed.